eZ/Publish/FieldType/Tweet/FormMapper.php

```diff
@@ -8,28 +8,82 @@
+    private $fieldTypeService;
+
+    public function __construct(FieldTypeService $fieldTypeService)
+    {
+        $this->fieldTypeService = $fieldTypeService;
+    }
+
+    /**
+     * @param FormInterface $fieldDefinitionForm
+     * @param FieldDefinitionData $data
+     * @throws \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
+     * @throws \Symfony\Component\Form\Exception\UnexpectedTypeException
+     * @throws \Symfony\Component\Form\Exception\LogicException
+     * @throws \Symfony\Component\Form\Exception\AlreadySubmittedException
+     */
     public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, FieldDefinitionData $data)
     {
         $fieldDefinitionForm
             ->add(
-                // Creating from FormBuilder as we need to add a DataTransformer.
+            // Creating from FormBuilder as we need to add a DataTransformer.
                 $fieldDefinitionForm->getConfig()->getFormFactory()->createBuilder()
                     ->create('authorList', TextType::class, [
                         'required' => false,
                         'property_path' => 'validatorConfiguration[TweetValueValidator][authorList]',
+                        'translation_domain' => 'eztweet_fieldtype',
                         'label' => 'field_definition.eztweet.authorList'
                     ])
                     ->addModelTransformer(new StringToArrayTransformer())
                     // Deactivate auto-initialize as we're not on the root form.
-                    ->setAutoInitialize(false)->getForm()
+                    ->setAutoInitialize(false)
+                    ->getForm()
+            );
+    }
+
+    /**
+     * @param FormInterface $fieldForm
+     * @param FieldData $data
+     * @throws \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
+     * @throws \Symfony\Component\Form\Exception\UnexpectedTypeException
+     * @throws \Symfony\Component\Form\Exception\LogicException
+     * @throws \Symfony\Component\Form\Exception\AlreadySubmittedException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     */
+    public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
+    {
+        $fieldDefinition = $data->fieldDefinition;
+        $formConfig = $fieldForm->getConfig();
+        $label = $fieldDefinition->getName($formConfig->getOption('mainLanguageCode')) ?: reset($names);
+        $fieldType = $this->fieldTypeService->getFieldType($fieldDefinition->fieldTypeIdentifier);
+        $fieldForm
+            ->add(
+                $formConfig->getFormFactory()->createBuilder()
+                    ->create(
+                        'value',
+                        TextType::class,
+                        [
+                            'required' => false,
+                            'label' => $label
+                        ]
+                    )
+                    ->setAutoInitialize(false)
```

Comment:
I would move this line after `addModelTransformer` and also add the comment "// Deactivate auto-initialize as we're not on the root form" to be consistent with `mapFieldDefinitionForm()`.